core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@hachikuji Yes, thanks. That's what I realized when I guess you were writing your last post :)
I'm going to look further into that last piece, which is how it would work in the new consumer, as you explained. I might bug you again for clarification. Thanks for bearing with me no this.